src/globus_sdk/_testing/data/flows/_common.py

```diff
@@ -123,6 +123,12 @@
     "flow_administrators": [
         "urn:globus:auth:identity:05d29dab-bd26-4510-9290-468972e8ac01"
     ],
+    "run_managers": [
+        "urn:globus:auth:identity:7d6064ef-5368-473a-b15b-e99c3561aa9b",
+    ],
+    "run_monitors": [
+        "urn:globus:auth:identity:57088a17-d5cb-4cfa-871a-c5cce48f2aec",
+    ],
```

Comment:
I know this is adding to existing art, not creating something new, but the fact that there are a _bunch_ of hardcoded IDs here and that they're matched across the other fixture data (`TWO_HOP_TRANSFER_RUN`) seems like something we should ease out of a bit.

Can we at least, for now, define these near the top of the file and then re-use the same constant, rather than carefully matching string literals? i.e.
```python
RUN_MANAGER_PRINCIPAL = f"urn:globus:auth:identity:{str(uuid.uuid1())}"
RUN_MONITOR_PRINCIPAL = ...
```
and then use those here and in the run doc below?

Reply:
@sirosen I believe I've addressed all the comments for this PR.